agent/bench-scripts/pbench_fio

```diff
@@ -304,24 +304,32 @@ function fio_process_options() {
 
 # Ensure the right version of the benchmark is installed
 function fio_install() {
-	if [ "$postprocess_only" == "n" ]; then
+	if [ -e $benchmark_bin ]; then
+		debug_log "[$script_name]$benchmark is already installed"
+		echo "$benchmark is already installed" 
+	elif [ "$postprocess_only" == "n" ]; then
 		if check_install_rpm $benchmark $ver; then
 			debug_log "[$script_name]$benchmark is installed"
 		else
 			debug_log "[$script_name]$benchmark installation failed, exiting"
 			exit 1
-		fi
-		if [ ! -z "$clients" ]; then
-			debug_log "verifying clients have fio installed"
-			echo "verifying clients have fio installed"
-			for client in `echo $clients | sed -e s/,/" "/g`; do
-				ssh $ssh_opts $client ${pbench_install_dir}/bench-scripts/pbench_fio --install &
-			done
-			wait
-		fi
-		if [ "$install_only" == "y" ]; then
-			exit 0
-		fi
+		fi 
+	fi
+	if [ ! -z "$clients" ]; then
+		debug_log "verifying clients have fio installed"
+		echo "verifying clients have fio installed"
+		for client in `echo $clients | sed -e s/,/" "/g`; do
+			if ( ssh $ssh_opts $clients "[ ! -e $benchmark_bin ]" ); then 
+				ssh $ssh_opts $client yum -y install $benchmark-$ver
```

Comment:
@atheurer if client is docker container / ose pod then having pbench installed in all of them is not necessary. Having pbench only on host system where pods/containers run and collect stats from host is sufficient 
The problem I had is actually not keeping my version of fio ( upstream ) and replacing it with pbench_fio fio if pbench_fio is run